src/views/CustomToken.vue

```diff
@@ -99,9 +99,11 @@
           <div class="form-group">
             <label for="decimals">{{ $t('pages.customToken.decimals') }}</label>
             <input
-              type="text"
-              v-model="decimals"
+              type="number"
+              v-model.trim="decimals"
+              @keypress="isNumber($event)"
               class="form-control form-control-sm"
+              :maxlength="2"
```

Comment:
@mpwanyi256 why are we using a max length of 2 here?

Reply:
@Jennievon This enables us to limit the value of decimal places to a maximum of 99.